### ISSUE TITLE
in_array type error: Second argument ( $haystack ) must be of type array, null given.

### ISSUE THREAD

**Issue by adinan-cenci** [juanantoniodelgado/StopWords#14]
## Overview
Happening under PHP 8.1

```
TypeError: in_array(): Argument #2 ($haystack) must be of type array, null given in vendor/juanantoniodelgado/stopwords/src/Cache.php on line 64 

#0 vendor/juanantoniodelgado/stopwords/src/Cache.php(64): in_array('pt', NULL)
#1 vendor/juanantoniodelgado/stopwords/src/Cache.php(39): StopWords\Cache->inHandlers('pt')
```

A downcast to array should fix it, me thinks.

**Comment by adinan-cenci** [comment]
but the actual problem is in the Indonesian language file, it reads `"handler"` instead of `"handlers"`

**Comment by adinan-cenci** [comment]
wait ... this seems to have been [fixed fixed 2 years ago](https://github.com/juanantoniodelgado/StopWords/commit/bba4ac0938b63dd70c07d7ef3a11d0c16df2d6b1)... then why composer installed it with the problem still there ? I am using 1.3

![image](https://github.com/user-attachments/assets/a34fc0f5-955e-4e00-a816-ae0cd32a15e2)

**Comment by juanantoniodelgado** [comment]
My apologies, but it seems that we never released a new version with the fix on Packagist. This has been fixed now by releasing [1.4](https://packagist.org/packages/juanantoniodelgado/stopwords#1.4).
